app/services/patient_deduplication/deduplicator.rb

```diff
@@ -184,6 +185,10 @@ def create_encounters_and_observables(patient)
       end
     end
 
+    def create_notifications(patient)
+      create_cloned_records!(patient, Notification, Notification.where(patient_id: latest_patient.id))
```

Comment:
We should probably also clone communications, ya?

Reply:
And do we discard communications when we discard notifications (when we discard a patient)?